blockbase/sass/elements/_links.scss

```diff
@@ -11,7 +12,7 @@ a {
 }
 
 //Links that appear in the main content area
-.block-editor-block-list__layout a, // Needed for the post area
+main.block-editor-block-list__layout a, // Needed for the post area
 .wp-block-post-content a {
```

Comment:
This looks good in the editor for the main content where I can see the underlines, but I still can't see the underlines on the front end, e.g. on the homepage:

<img width="652" alt="image" src="https://user-images.githubusercontent.com/1645628/160587405-0ab76d6e-9447-4333-9f1d-9e67bb10cbfd.png">

Looks like we just need to change this selector to `.wp-block-post a` instead of `.wp-block-post-content a`.

Reply:
Included suggested changes and following is the UI.

<img width="694" alt="image" src="https://user-images.githubusercontent.com/1935113/161012115-1f8c728b-aaa3-4079-9baf-b319fb657804.png">

Reply:
Awesome, looks good to me! 🎉